### ISSUE TITLE
Reproducibility with Docker

### ISSUE THREAD

**Issue by huguesfontenelle** [agiess/PREN-TIS#2]
Hi
To ensure reproducibility or facilitate adoption, it would be nice to include a Dockerfile, and possibly push the image to [DockerHub](https://hub.docker.com/).
What do you think?

One option is [bioconda](https://bioconda.github.io/) which has recipes for all the tools, except perhaps [h20](http://docs.h2o.ai/).
Alternatively, build from [r-base](https://hub.docker.com/_/r-base/), but then run installs for both `perl` and `samtools`.

**Comment by agiess** [comment]
Thanks, this is a good suggestion. I did not go down this route initially because of problems using docker on our infrastructure. I would like to have this implemented though, so I am looking into solutions.

**Comment by agiess** [comment]
I have added a Dockerfile and linked the repository to [dockerhub](https://cloud.docker.com/repository/docker/adamgiess/pren_tis). I have also added a testing dataset to help with deployment.

Future improvements:
1) I started from a h2o + tidyverse image. This saves time, but results in an slightly larger image than installing from base-r.
2) In hindsight for this deployment method using git clone in the Dockerfile is a bit counter intuitive.
